tests/modules/local/maptotsv/main.nf.test

```diff
@@ -0,0 +1,289 @@
+nextflow_process {
+
+    name "Test Process MAP_TO_TSV"
```

Comment:
Could you add additional test cases to this file/make sure they are handled in the code:

1. Instead of empty string `""` as missing values could you test the case of `null` as missing values (in case `null` ever is returned by Nextflow for one of the metadata values).
2. Could you test a case of one or more headers being `null` or empty (in case one of the header names was set to `null`).
3. Could you test the case where the number of columns for headers and rows are mismatched (e.g., 2 headers, but 1 value in a row)? I would expect the process to raise an exception/return an error in this case. I know this won't happen with your current code, but it's good to have a check in case refactoring down the line led to this situation.
4. Could you test the case of passing all empty data (empty headers, empty rows).
5. Could you test the case of the function returning all empty data (e.g., all columns are removed)?

Reply:
Added in 08ee7bd9b5de0e0b3f72a9b4320ab902efd7be16 and 1dd07cbec1d626db4c8cc0e63810cc5110c56ba1

Note that item 4 causes item 5, so they're tested at the same time.